README.md

```diff
@@ -79,9 +79,10 @@ zone.  This role has a trust relationship with the users account.
 | [aws_route53_record.crossfeed_prod_ses_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.crossfeed_staging_A](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.crossfeed_staging_AAAA](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
+| [aws_route53_record.crossfeed_staging_ACME_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.crossfeed_staging_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.crossfeed_staging_MX](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
-| [aws_route53_record.crossfeed_staging_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
+| [aws_route53_record.crossfeed_staging_TXT1](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

Comment:
This change should be reverted to match the change in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/80/commits/dfe350d7cf8a4f5423162c02cc356f6bc9560b93:
```suggestion
| [aws_route53_record.crossfeed_staging_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

Reply:
Resolved in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/80/commits/70f7417059899955ae75429316fa1bd73e9dc148.